plugins/nvim-lsp/options/servers.nix

```diff
@@ -92,6 +92,11 @@ let
       serverName = "rust_analyzer";
       packages = [ cargo rust-analyzer ];
     };
+    svelte-language-server = {
+      languages = "svelte";
+      serverName = "svelteserver";
```

Comment:
The correct server name appears to be `svelte` not `svelteserver`.  It is listed as such in [lspconfig](https://github.com/neovim/nvim-lspconfig/tree/master/lua/lspconfig/server_configurations)